### PR TITLE
Add new feature Airplane.dev CLI

### PR DESCRIPTION
**Pull request by GMkonan**
[Airplane.dev](https://www.airplane.dev/)

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -353,3 +353,8 @@
   contact: https://github.com/ksh5022/devcontainer-features/issues
   repository: https://github.com/ksh5022/devcontainer-features
   ociReference: ghcr.io/ksh5022/devcontainer-features
+- name: GMkonan devcontainer features
+  maintainer: GMkonan
+  contact: https://github.com/GMkonan/feature/issues
+  repository: https://github.com/GMkonan/feature
+  ociReference: ghcr.io/GMkonan/feature
```
Comment (samruddhikhandale) [inline _data/collection-index.yml]:
```suggestion
  ociReference: ghcr.io/gmkonan/feature
```
Hi 👋  The ORAS spec requires the `ociReference` to be in lower case. Can we update here, thanks?
The supporting tools do help convert the case, however, its better to sync as much as possible.

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
This is awesome, thanks! Left one minor comment

File: _data/collection-index.yml (hunk 1 above)
Comment (GMkonan) [reply _data/collection-index.yml]: Sure! Already updated.

**Review by samruddhikhandale — APPROVED** [review]
Nice, thanks! 🚀

**Pull request merged**